### ISSUE TITLE
Changes to polite ARIA Live Region triggered by button click are only announced by JAWS 2024 when Virtual Cursor is off (Firefox)

### ISSUE THREAD

**Issue by MarcSolo** [FreedomScientific/VFO-standards-support#866]
### Summary

JAWS 2024.2405.83 does not announce changes to a polite ARIA Live Region in Firefox 129.0.2 unless you first turn off the Virtual Cursor.

Test steps:
1. Turn screen reader on (make sure Virtual Cursor is on/Insert+Z)
2. Open [carousel example in the WAI tutorial](https://www.w3.org/WAI/tutorials/carousels/working-example/%20and%20here%20are%20the%20results) 
3. Navigate to the "Next item" button
4. Activate button
5. Observe screen reader announcement

### Expected result

JAWS announces changes to ARIA Live Region, "Item X of Y"

### Actual result

JAWS does not announce changes to ARIA Live Region unless you turn off the Virtual Cursor first and then activate the button.

### Example

[carousel example in the WAI tutorial](https://www.w3.org/WAI/tutorials/carousels/working-example/%20and%20here%20are%20the%20results)

### Additional Information

Cannot reproduce with JAWS and Chrome 127.0.6533.122

Cannot reproduce with NVDA 2024.1 and Chrome 127.0.6533.122 or Firefox 129.0.2

#### JAWS version and build number

JAWS 2024.2405.83
 
#### Operating System and version

Windows 10.0.19045 

#### Browser and version:
Firefox 129.0.2

**Comment by MarcSolo** [comment]
The video attached is a screen cast highlighting the difference in JAWS behavior activating the next item button with the Virtual Cursor on and off. JAWS does not announce the changes to the Live Region (e.g. "Item 2 of 3") when activating the button with the Virtual Cursor on. It does when the Virtual Cursor is turned off by pressing Insert+Z
https://github.com/user-attachments/assets/c10b8ce4-a972-4663-be75-fde0f6b60a28
